Octokit/Clients/IIssuesClient.cs

```diff
@@ -52,7 +52,7 @@ public interface IIssuesClient
         /// <param name="number">The issue number</param>
         [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Get",
              Justification = "Method makes a network request")]
-        Task<Issue> Get(string owner, string name, int number);
+        Task<Issue> Get(string owner, string name, long number);
```

Comment:
Same thing here - this `number` used in this interface is not the one overflowing. Can we undo the changes in this file to simplify the diff?